src/sumo_gym/utils/helloWorld.net.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You can use the xml file under asset/data instead. That's a runnable sumo env created via Netedit.